pkg/registry/apps/controllerrevision/storage/storage_test.go

```diff
@@ -160,7 +160,10 @@ func newStorage(t *testing.T) (*REST, *etcd3testing.EtcdTestServer) {
 		Decorator:               generic.UndecoratedStorage,
 		DeleteCollectionWorkers: 1,
 		ResourcePrefix:          "controllerrevisions"}
-	storage := NewREST(restOptions)
+	storage, err := NewREST(restOptions)
+	if err != nil {
+		t.FailNow()
```

Comment:
My previous comment seems skipped on the above comment.
Here also needs `t.Errorf("unexpected error: %v", err)` before `t.FailNow()`
I am putting every point where necessary to add the above line.